pkg/network/usm/istio.go

```diff
@@ -115,11 +115,14 @@ func newIstioMonitor(c *config.Config, mgr *manager.Manager) *istioMonitor {
 	}
 
 	procRoot := kernel.ProcFSRoot()
+	if c.EnvoyPath != "" {
+		envoyCmd = []byte(c.EnvoyPath)
```

Comment:
Don't override the global
although no bad impact will happen (as we have a single instance of istio monitoring)
but, that's not future proof
I can image having other "envoy monitors", and overriding envoyCmd can cause bad outcomes